docs/squadcast.md

```diff
@@ -0,0 +1,68 @@
+# Use with Squadcast
+
+Syntax:
+
+`export SQUADCAST_REFRESH_TOKEN=<YOUR_SQUADCAST_REFRESH_TOKEN>`
+
+```
+terraformer import squadcast --resources=<SERVICE_NAMES> --region=SQUADCAST_REGION
+```
+
+Examples:
+
+- `Import User Resource`
+
+```
+terraformer import squadcast --resources=user --region=us
+```
+
+- `Import Squad Resource`
+
+```
+terraformer import squadcast --resources=team --region=us --team-name="Default Team"
+```
+
+- `Import Resource by providing refresh-token as a flag`
+
+```
+terraformer import squadcast --resources=team --region=us --team-name="Default Team" --refresh-token=YOUR_REFRESH_TOKEN
+```
+
+### Flags:
+
+- `--team-name`
+
+  - Required for the following resources:
+    - squad
+    - service
+    - escalation_policy
+    - team_member
+    - team_roles
+    - slo
+    - runbook
+
+- `--region`
+  - Supported Values:
+    - `us`
+    - `eu`
```

Comment:
fix the aligment

Reply:
It looks something like this. 
![image](https://user-images.githubusercontent.com/110392771/187366627-403764ac-b37f-4715-88b8-6b1916bbd67e.png)